python/ml4ir/applications/classification/model/classification_model.py

```diff
@@ -251,13 +251,77 @@ def predict(
         # than a list on numpy arrays
         predictions_df[self.output_name] = [x for x in predictions_]
         if logs_dir:
-            np.set_printoptions(formatter={'all':lambda x: str(x.decode('utf-8')) if isinstance(x, bytes) else str(x)},
-                                linewidth=sys.maxsize,
-                                threshold=sys.maxsize,  # write the full vector in the csv not a truncated version
-                                legacy="1.13")  # enables 1.13 legacy printing mode
+            # np.set_printoptions(formatter={'all':lambda x: str(x.decode('utf-8')) if isinstance(x, bytes) else str(x)},
+            #                     linewidth=sys.maxsize,
+            #                     threshold=sys.maxsize,  # write the full vector in the csv not a truncated version
+            #                     legacy="1.13")  # enables 1.13 legacy printing mode
+
+
+            # np.set_printoptions(formatter={
+            #     'all': lambda x: str(x.decode('utf-8', errors='ignore')) if isinstance(x, bytes) else str(x)
+            # }, linewidth=sys.maxsize, threshold=sys.maxsize, legacy="1.13")
+
+            # for col in predictions_df.columns:
+            #     if isinstance(predictions_df[col].values[0], bytes):
+            #         predictions_df[col] = predictions_df[col].str.decode('utf8')
```

Comment:
@mohazahran Can you remove unused code?

Reply:
I left it commented as it seems it really affect the java integration test. I will add comments to reflect that.